installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -60,27 +51,69 @@ include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+* xref:../../[Mirror registry update examples]
 * xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#oc-mirror-image-set-examples_installing-mirroring-disconnected[Image set configuration examples]
 * xref:../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[Using the OpenShift Update Service in a disconnected environment]
 
+[id="setting-incremental-mirrorring"]
+== Setting up for incremental mirroring
+
+In most cases, you are mirroring images to install, upgrade and maintain Openshift in a disconnected environment. Because the volume of the image set needed to install or upgrade Openshift can be substantial, oc-mirror keeps metadata about previously mirrored images in its storage backend in order to reduce the volume of the following image sets to the recent images only.
+
+Therefore, you must specify a storage backend in the image set configuration file. The oc-mirror plugin stores metadata in this storage backend during image set creation. This storage backend can be:
+* A local directory 
++
+[NOTE]
+====
+This is not recommended for enterprise use cases as it doesn't scale as well as the registry based storage backend.
+====
++
+[source,terminal]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:                                                    
+  local:
+    path: <oc-mirror-workspace>
+----
+* or a registry that supports Docker v2-2
+[source,terminal]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:                                                    
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata                 
+    skipTLS: false
+----
+
+When mirroring content to a given location on your mirror registry with the oc-mirror plugin: 
+- Ensure you always use the same storage backend and the same image set configuration for a given mirror registry location, so that the metadata remains consistent.
+- Do not attempt to push or prune images of that mirror registry location manually, without using oc-mirror.
+- Do not delete or modify the metadata that is generated by the oc-mirror plugin.
+- It is recommended to keep a backup of the storage backend. When mirroring content to a given location on your mirror registry with the oc-mirror plugin.
+
 [id="mirroring-image-set"]
 == Mirroring an image set to a mirror registry
 
-You can use the oc-mirror CLI plugin to mirror images to a mirror registry in a xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#mirroring-image-set-partial[partially disconnected environment] or in a xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#mirroring-image-set-full[fully disconnected environment].
+When you populate your mirror registry with OpenShift Container Platform images, you can follow two scenarios:
```

Comment:
I am not sure if it is okay to call `openshift container platform images` as having this just refers to only platform images and additional / operator images does not come into picture here.

instead of `you can follow two scenarios` should you say Below listed are the two ways in which images could be mirrored to your registry` ?

Reply:
Does this sound fine:
To populate your mirror registry with OpenShift Container Platform images, you can use one of the following two scenarios: